pkgs/by-name/ni/niri/package.nix

```diff
@@ -20,19 +20,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "niri";
-  version = "0.1.7";
+  version = "0.1.8";
 
   src = fetchFromGitHub {
     owner = "YaLTeR";
     repo = "niri";
-    rev = "v${version}";
-    hash = "sha256-EdlOGL9RdO77HnZxv2UpPwrJdFH8qPrPSRtkBBV167s=";
+    rev = "refs/tags/v${version}";
```

Comment:
Why are we changing this? Is there a clear benefit?

Reply:
There is no difference between `v${version}` and `refs/tags/v${version}`, it only adds visual clutter.